database/src/main/java/info/metadude/android/eventfahrplan/database/sqliteopenhelper/SessionsDBOpenHelper.java

```diff
@@ -8,37 +8,36 @@
 
 import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable;
 import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable.Columns;
-import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable.Defaults;
-import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable.Values;
+import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.SessionsTable.Indices;
 
 public class SessionsDBOpenHelper extends SQLiteOpenHelper {
 
-    private static final int DATABASE_VERSION = 9;
+    private static final int DATABASE_VERSION = 10;
```

Comment:
You might as well reset this to 1 if you also change the database name.

Reply:
:heavy_check_mark: Done in be9c1bffdb14dedf966f5f69d1c670b56fe3d04c. Thanks.